File: Programas em Java/03 - Laço While e For/Script1.java
/*
 * Faca um algoritmo que determine o maior entre N numeros. A condicao de
 * parada é a entrada de um valor 0, ou seja, o algoritmo deve ficar calculando
 * o maior até que a entrada seja igual a 0 (Zero).
 */
package geek;

import java.util.Scanner;

public class Script1 {

	public static void main(String[] args) {
		//variáveis
		int n, maior = 0;
		Scanner teclado = new Scanner(System.in);
		
		//Entrada
		System.out.println("Informe um número: ");
		n = teclado.nextInt();
		
		while(n != 0) {
			if(n > maior) {
				maior = n;
			}
			System.out.println("Informe um número: ");
			n = teclado.nextInt();
		}
		System.out.printf("O maior número é %d", maior);
		teclado.close();

	}

}

File: Programas em Python/03 - Laço While e For/Script1.py
# Faca um algoritmo que determine o maior entre N numeros. 
# A condicao de parada é a entrada de um valor 0, ou seja, 
# o algoritmo deve ficar calculando o maior até que a 
# entrada seja igual a 0 (Zero).

#variáveis
maior = 0
#entrada
n = int(input("Informe um número: "))
while n != 0:
    if n > maior:
        maior = n
    n = int(input("Informe um número: "))
print("O maior número é {0}".format(maior))
